### PR TITLE
Add the Sim09 Version - Gauss Version correspondance

### PR DESCRIPTION
**Pull request by AndyLHCb**
Also how to find the production IDs of existing samples

### CODE REVIEW

File: second-analysis-steps/simulation-intro.md [hunk 1]
```diff
@@ -20,3 +20,28 @@ Gauss is the LHCb simulation framework which manages the creation of simulated e
 3. The number of generated events refers to the number entering step 4 above, so those passing the generator level cuts. __Not__ the number of events produced by the `ProductionTool` in the first step.
 
 {% endcallout %}
+
+## Choosing your Gauss Version
+
+Each release of Gauss v49 represents a different release of Sim09. You may want to run an older version of Gauss to verify something in a pre-existing MC dataset or for any number of other reasons. Here is a table matching each release to each Sim09 realease:
```
Comment (vlisovsk) [inline second-analysis-steps/simulation-intro.md:26]: "matching each release to each Sim09 realease" -> suggest changing one of 'release' to 'version'

File: second-analysis-steps/simulation-intro.md [hunk 2]
```diff
@@ -20,3 +20,28 @@ Gauss is the LHCb simulation framework which manages the creation of simulated e
...
+| Sim09j | v49r18 |
+| Sim09k | v49r20 |
+
+Any Gauss release not listed here corresponds to the lower of the Sim09 versions it is between (e.g. Gauss v49r9 is Sim09c)
+
+Whilst these act as guidelines, to find the version of Gauss used for a specific production you can run this command:
+`lb-dirac dirac-bookkeeping-production-information <prodID>`.
+Here `<prodID>` represents your production number, this is a 5 or 6 digit number. If you don't know this, it can be found by using this command:
+`lb-dirac dirac-bookkeeping-decays-path <Dec Number>`.
+Here the production ID will be the last number given in each line. Note that you may need to use ProdID-1 for flagged samples or ProdID-2 for filtered samples since the generation, filtering and stripping are done in different steps
```
Comment (vlisovsk) [inline second-analysis-steps/simulation-intro.md:47]: please add period at the end of this sentence

**Comment by vlisovsk** [comment]
Maybe another point: I have not checked if the lessons cover that already, but should we explain what is Sim10 and what are the upgrade simulation tunes?

**Comment by AndyLHCb** [comment]
> Maybe another point: I have not checked if the lessons cover that already, but should we explain what is Sim10 and what are the upgrade simulation tunes?

Were you envisioning something short like "Just fyi, Sim10 exists corresponding to Gauss v55+ and will be used for Run 3 simulations with the updated detector geometry however this lesson will focus on running Gauss for Runs 1 and 2"? Or did you mean something more in-depth?

**Comment by vlisovsk** [comment]
What about something like:
```
Once in a few years, major Sim versions are released, which involve newer versions of GEANT4, Pythia8 and other software, as well as major fixes to the detector description. Prior to production release, an extensive testing is performed by the Simulation group. The next major release is Sim10 (Gauss versions above v52) which is currently under validation. It will be used for both Run 1-2 and upgrade simulations in the future. The current Sim09 also supports production for upgrade. Sim versions suitable for upgrade simulations are marked with "Up" label, e.g. `Sim10-Up08`. 
```
(feel free to reprhase or correct)

**Comment by AndyLHCb** [comment]
That's perfect! I think I'll just add that verbatim

**Pull request merged**